andela/pulse-mobile-app
# [#156371592] Test Set Up

### What does this PR do?
Sets Up testing environment for the Project.

### Description of Task to be completed?
 - Set Up testing environment.
 - Added folder structure for test files.
 - Add placeholder tests for the components.

### How should this be manually tested?
On the terminal, navigate to application and run `yarn test`. This should run the tests successfully.

### What are the relevant pivotal tracker stories?
[#156371592](https://www.pivotaltracker.com/story/show/156371592)

### Screenshots (if appropriate)
![screen shot 2018-04-04 at 17 42 23](https://user-images.githubusercontent.com/17447937/38314729-93cbd880-382f-11e8-9af7-b1ff9e0c9f61.png)
